gatsby-config.js

```diff
@@ -128,7 +128,7 @@ module.exports = {
                     }),
             },
         },
-        'gatsby-plugin-remove-serviceworker',
```

Comment:
Should be listed after the manifest I think? so the manifest can be included in the service worker.

https://www.gatsbyjs.com/plugins/gatsby-plugin-offline/

Reply:
Lighthouse complains about this one as well:

![image](https://user-images.githubusercontent.com/53067656/95721629-7828a980-0ca5-11eb-8e88-9f5fc7fd5c8d.png)